src/feed_parser.c

```diff
@@ -174,15 +177,13 @@ feed_parse (feedParserCtxtPtr ctxt)
 {
 	xmlNodePtr	xmlNode = NULL, htmlNode = NULL;
 	xmlDocPtr	xmlDoc, htmlDoc;
-	gboolean	success = FALSE;
+	gboolean	autoDiscovery = FALSE, success = FALSE;
 	gboolean	htmlParsing = FALSE;
```

Comment:
On the topic of warnings, the `htmlParsing` variable is unused.

Reply:
Fixed